src/core2/graph.js

```diff
@@ -79,6 +80,18 @@ export interface PluginHandler<NR: NodeReference, NP: NodePayload> {
 
 export type Plugins = $ReadOnlyArray<PluginHandler<any, any>>;
 
+type NodesSortedByStringifiedAddress = (
+  | {|+address: Address|}
+  | {|+payload: any, +pluginName: string|}
+)[];
+export type GraphJSON = {|
+  +nodes: NodesSortedByStringifiedAddress,
+  +edges: AddressMapJSON<IndexedEdge>,
+|};
+
+const COMPAT_TYPE = "sourcecred/sourcecred/Graph";
```

Comment:
Wondering what you think of just `sourcecred/Graph`? As in, the organization SourceCred will only ever publish one "Graph". Then it's actually portable if we refactor our repository structure.

Reply:
The principle of organizations owning compat types is fine with me. In
this particular case, it might be okay: “graph” is a pretty big class of
thing to promise to only ever create one of, but (a) maybe that’s true
for SourceCred, and (b) we’re well before the point where we provide any
actual compat guarantees. :-)

I’m happy to make this change if you like.